keras/objectives.py

```diff
@@ -63,6 +63,36 @@ def cosine_proximity(y_true, y_pred):
     y_pred = K.l2_normalize(y_pred, axis=-1)
     return -K.mean(y_true * y_pred, axis=-1)
 
+# The actual pinball/ tilted loss function for quantile regression
+def tilted_loss(q, y, f):
+    """
+    Pinball loss function used in quantile regression where q is the quantile.
+    """
+    e = (y-f)
+    return K.mean(q*e + K.clip(-e, K.epsilon(), np.inf), axis=-1)
```

Comment:
I fixed as much as I could with the automatic fixer as pointed out on Contributions.md but I'm not too familiar with PEP8. Is there a particular convention that I have violated?